### ISSUE TITLE
docker 17.03 swarm: Error response from daemon: Invalid filter 'scope'

### ISSUE THREAD

**Issue by richardjq** [containous/traefik#2305]
When running traefik 1.4 on a docker swarm that has docker version 17.03 you get error message:

```
time="2017-10-24T07:04:29Z" level=debug msg="Provider connection established with docker 17.03.1-ce (API 1.27)" 
time="2017-10-24T07:04:29Z" level=debug msg="Failed to network inspect on client for docker, error: %sError response from daemon: Invalid filter 'scope'" 
time="2017-10-24T07:04:29Z" level=error msg="Failed to list services for docker swarm mode, error Error response from daemon: Invalid filter 'scope'" 
time="2017-10-24T07:04:29Z" level=error msg="Provider connection error Error response from daemon: Invalid filter 'scope', retrying in 504.818146ms" 
```

This appears to be because 17.03 doesn't have that filter option:

```bash
$ docker --version
Docker version 17.03.1-ce, build c6d412e
$ docker network ls --filter scope=swarm
Error response from daemon: Invalid filter 'scope'
```

versus 17.09:

```bash
$ docker --version
Docker version 17.09.0-ce, build afdb6d4
$ docker network ls --filter scope=swarm
NETWORK ID          NAME                DRIVER              SCOPE
```

It appears this is being set in `provider/docker/docker.go`, line 870:

```go
869 | networkListArgs := filters.NewArgs()
870 | networkListArgs.Add("scope", "swarm")
871 |  
872 | networkList, err := dockerClient.NetworkList(ctx, dockertypes.NetworkListOptions{Filters: networkListArgs})
```

I understand that 17.03 isn't the most current version of docker, but for those of using docker/traefik in the enterprise it would be helpful if traefik supported at least the docker versions released within the past year.

Thanks.

**Comment by mmatur** [comment]
I close this issue for a duplicate of #2297

Closed by #2301.

This will be available in 1.4.1 version

**Comment by richardjq** [comment]
Awesome! Thank you.
